sample/UseMaven/GNAdGoogleRewardMediationSample/app/src/main/AndroidManifest.xml

```diff
@@ -56,5 +54,10 @@
             android:name="applovin.sdk.key"
             android:value="YOUR_APPLOVIN_SDK_KEY_HERE" />
 
+        <!-- TikTok -->
```

Comment:
In the sample app, implementation of Tiktok library is added, but it is not necessary.

Reply:
@hironobu-yazaki  TikTok was added automatically after I executed this command `./Shells/create-github-projects.sh` from `ssp-android-sdk-samples`

Do I need to remove it?

Reply:
Yes, please remove it.

Reply:
@hironobu-yazaki Thank you. I already removed it on my latest commit